src/lang-promql/complete/hybrid.ts

```diff
@@ -302,29 +284,7 @@ export class HybridComplete implements CompleteStrategy {
       from: from,
       to: to,
       options: options,
+      filterDownOn: /^[a-zA-Z0-9_:]+$/,
```

Comment:
Maybe we should make it optional ?

Reply:
Hm we could, maybe in a followup version? We also have to rethink the characters maybe, since they would depend on the type of object being completed...

Reply:
Yeah ok. It will give us time to see if it's a good idea or something that should stay in the garbage 1 am Idea hahaha